docs/devsite-help/retries.md

```diff
@@ -0,0 +1,32 @@
+# Configuring Retries and Timeouts
```

Comment:
Look at the one in call-settings.md and maybe modify it if there's something missing from there, plus, add the sample code that we already have?